test/powershell/Language/Parser/Parser.Tests.ps1

```diff
@@ -882,10 +882,10 @@ foo``u{2195}abc
             @{ Script = "0b0"; ExpectedValue = "0"; ExpectedType = [int] }
             @{ Script = "0b10"; ExpectedValue = "2"; ExpectedType = [int] }
             @{ Script = "-0b10"; ExpectedValue = "-2"; ExpectedType = [int] }
-            @{ Script = "0b11111111"; ExpectedValue = "-1"; ExpectedType = [int] }
-            @{ Script = "0b1111111111111111"; ExpectedValue = "-1"; ExpectedType = [int] }
-            @{ Script = "0b11111111111111111111111111111111"; ExpectedValue = "-1"; ExpectedType = [int] }
-            @{ Script = "0b1111111111111111111111111111111111111111111111111111111111111111"; ExpectedValue = "-1"; ExpectedType = [long] }
+            @{ Script = "0b11111111"; ExpectedValue = "255"; ExpectedType = [int] }
+            @{ Script = "0b1111111111111111"; ExpectedValue = "65535"; ExpectedType = [int] }
+            @{ Script = "0b11111111111111111111111111111111"; ExpectedValue = "4294967295"; ExpectedType = [int] }
+            @{ Script = "0b1111111111111111111111111111111111111111111111111111111111111111"; ExpectedValue = "18446744073709551615"; ExpectedType = [long] }
```

Comment:
these two shouldn't change, based on the WG feedback and the code changes you've got here ^^